testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/AbstractClientPoliciesTest.java

```diff
@@ -558,195 +803,843 @@ protected void deleteClientDynamically(String clientId) throws ClientRegistratio
+        }
+
+        public ClientProfileBuilder createProfile(String name, String description, Boolean isBuiltin, List<Object> executors) {
+            if (name != null) {
+                profileRep.setName(name);
+            }
+            if (description != null) {
+                profileRep.setDescription(description);
+            }
+            if (isBuiltin != null) {
+                profileRep.setBuiltin(isBuiltin);
+            } else {
+                profileRep.setBuiltin(Boolean.FALSE);
+            }
+            if (executors != null) {
+                profileRep.setExecutors(executors);
+            } else {
+                profileRep.setExecutors(new ArrayList<>());
+            }
+            return this;
+        }
+
+        public ClientProfileBuilder addExecutor(String providerId, Object config) {
+            ObjectMapper mapper = new ObjectMapper();
+            String configString = null;
+            if (config == null) {
+                configString = "{}";
+            } else {
+                try {
+                    configString = mapper.writeValueAsString(config);
+                } catch (JsonProcessingException e) {
+                    fail();
+                }
+            }
+            String executorJson = (new StringBuilder())
+                    .append("{\"")
+                    .append(providerId)
+                    .append("\":")
+                    .append(configString)
+                    .append("}")
+                    .toString();
+            JsonNode node = null;
+            try {
+                node = mapper.readTree(executorJson);
+            } catch (JsonProcessingException e) {
+                fail();
+            }
+            profileRep.getExecutors().add(node);
+            return this;
+        }
+
+        public ClientProfileRepresentation toRepresentation() {
+            return profileRep;
+        }
+
+        public String toString() {
+            ObjectMapper mapper = new ObjectMapper();
+            String profileJson = null;
+            try {
+                profileJson = mapper.writeValueAsString(profileRep);
+            } catch (JsonProcessingException e) {
+                e.printStackTrace();
+                fail();
+            }
+            return profileJson;
+        }
+    }
+
+    // Client Profiles - Executor CRUD Operations
 
-    protected String createPolicy(String name, String providerId, String subType, List<String> conditions, List<String> executors) {
-        ComponentRepresentation component = createComponentInstance(name, providerId, ClientPolicyProvider.class.getName(), subType);
-        component.getConfig().put(DefaultClientPolicyProviderFactory.CONDITION_IDS, conditions);
-        component.getConfig().put(DefaultClientPolicyProviderFactory.EXECUTOR_IDS, executors);
-        return createComponent(component);
+    protected Object createHolderOfKeyEnforceExecutorConfig(Boolean isArgument) {
```

Comment:
Typo: should be `isAugment` . This applies to few other places on this class

Reply:
@mposolda Yes, I will fix these typos.